### PR TITLE
Updated conf.py to support activities and replaced deprecated stylesheet with add_css_file

### PR DESCRIPTION
**Pull request by EmosewaMC**
Changed base explorer url in the python link generator to point to explorer.lu and also added activity support to link to explorer.lu

### CODE REVIEW

**Review by Xiphoseer — APPROVED** [review]
FYI: This is the kind of PR I was referring to in Discord, where one part (update the URL) ist a sort of precondition for other stuff, while the deprecated function requires and entirely different kind of review, i.e. whether we are actually on a version new enough for this change.

Requirements.txt just says `<2` so we should indeed be using the latest 1.x in builds

**Comment by EmosewaMC** [comment]
> FYI: This is the kind of PR I was referring to in Discord, where one part (update the URL) ist a sort of precondition for other stuff, while the deprecated function requires and entirely different kind of review, i.e. whether we are actually on a version new enough for this change.
> 
> Requirements.txt just says `<2` so we should indeed be using the latest 1.x in builds

Yeah I realized this when I pushed the commit to the branch and should have waited until the first one was completed before working on the next part.  In the future what I'll do is just save the changes locally and wait for one pr to get merged first before merging more.  Thank you for being understanding while I learn more about working with others and with learning what changes should be in a pr and which ones need a separate PR due to them being entirely different changes.

**Pull request merged**
